system/jlib/jptree-attrvalues.hpp

```diff
@@ -0,0 +1,770 @@
+    "_",
+    "/",
+    ".",
+    "",
+    "*",
+    "\\,",
+    "0 secs",
+    "-1"
+    "1:1",
+    "1:M",
+    "* * * * *",
+    "* 2 * * *",
+    "* 3 * * *",
+    "Access to cluster topology",
+    "Access to despraying of files",
+    "Access to DFU",
+    "Access to DFU exceptions",
+    "Access to DFU workunits",
+    "Access to DFU XRef",
+    "Access to ECL direct service",
+    "Access to ECL Direct service",
+    "Access to ESDL configuration service",
+    "Access to files in dropzone",
+    "Access to machine information",
+    "Access to own workunits",
+    "Access to permissions for file scopes",
+    "Access to roxie control commands",
+    "Access to SMC service",
+    "Access to SNMP metrics information",
+    "Access to spraying files",
+    "Access to thor queues",
+    "Access to upload files to dropzone",
+    "Access to workunits owned by others",
+    "Access to WS ECL service",
+    "Access to WS Logging service",
+    "Action",
+    "Action #1",
+    "Action #2",
+    "active",
+    "activities",
+    "Aggregate",
+    "Alias",
+    "any",
+    "anyone",
+    "Application",
+    "archive",
+    "ASSERT",
+    "Associated",
+    "atftpd",
+    "Attr",
+    "Begin Loop",
+    "blank",
+    "blocked",
+    "Blocked by fileservice activity: Copy",
```

Comment:
yes I guess an edge case, even though it ranked highly - probably anomaly of regression suite.
I'll remove.